tests/test_meshing_workflow.py

```diff
@@ -741,3 +741,11 @@ def test_modified_workflow(new_mesh_session):
         task_display_names.append(name)
 
     assert set(task_display_names) == task_object_display_names
+
+
+def test_nonexistent_attrs(new_mesh_session):
+    meshing = new_mesh_session
+    assert not hasattr(meshing.workflow, "xyz")
+    with pytest.raises(AttributeError) as msg:
+        meshing.workflow.xyz()
```

Comment:
```suggestion
        meshing.workflow.xyz
```
The test is that access of `xyz` is sufficeint to raise an exception.